invenio_app_ils/documents/utils.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
```

Comment:
why not naming this files `covers_builder` or something instead of utils?
And I think it should not be inside `Document`, because it should be used for Series and Literature too.